contrib/database/sql/option.go

```diff
@@ -262,3 +263,12 @@ func WithDBMPropagation(mode tracer.DBMPropagationMode) Option {
 		cfg.dbmPropagationMode = mode
 	}
 }
+
+// WithDBStats enables polling of DBStats metrics on the registered DB
+// ref: https://pkg.go.dev/database/sql#DBStats
+// These metrics are submitted to Datadog and are not billed as custom metrics
```

Comment:
> not billed as custom metrics

How are we ensuring this?

Reply:
Haven't started this process just yet, but there's a process in place for this - we add the metric names to some list of OOTB metrics - Andrew shared this doc: https://datadoghq.atlassian.net/wiki/spaces/APW/pages/2524578676/Publishing+crawler+and+inbound+webhook+integrations